pkg/labels/labels.go

```diff
@@ -370,10 +381,17 @@ func parseSource(str string) (src, next string) {
 	sourceSplit := strings.SplitN(str, ":", 2)
 	if len(sourceSplit) != 2 {
 		next = sourceSplit[0]
-		if strings.HasPrefix(next, common.ReservedLabelKey) {
-			src = common.ReservedLabelSource
-			next = strings.TrimPrefix(next, common.ReservedLabelKey)
+		if strings.HasPrefix(next, common.BaseLabelSourceExtPrefix) {
```

Comment:
I'm convinced that this will totally confuse users. Even I don't get it. I thought this was entirely internal. If it is internal and you always attach a prefix, you always know that the first `word.` is the prefix.